akka-actor-typed/src/main/scala/akka/actor/typed/javadsl/AbstractBehavior.scala

```diff
@@ -32,7 +32,7 @@ import akka.util.OptionVal
  *
  * @see [[Behaviors.setup]]
  */
-abstract class AbstractBehavior[T](context: ActorContext[T]) extends ExtensibleBehavior[T] {
+abstract class AbstractBehavior[T](private val context: ActorContext[T]) extends ExtensibleBehavior[T] {
```

Comment:
Confusingly this actually exposes a public accessor for `context`.  Leaving it as a constructor parameter only is probably better.  Here's a minimise example bytecode code diff:

```diff
diff --git Foo.class Foo.class
index 77d1b35..2306501 100644
--- Foo.class
+++ Foo.class
@@ -4,10 +4,16 @@ public class Foo {
   private final java.lang.String s;
     descriptor: Ljava/lang/String;

-  public java.lang.String getString();
+  private java.lang.String s();
     descriptor: ()Ljava/lang/String;
     Code:
 ##: getfield      ###                 // Field s:Ljava/lang/String;
+##: areturn
+
+  public java.lang.String getString();
+    descriptor: ()Ljava/lang/String;
+    Code:
+##: invokespecial ###                 // Method s:()Ljava/lang/String;
 ##: areturn

   public Foo(java.lang.String);
diff --git Foo.scala Foo.scala
index b2f73b3..fc28b17 100644
--- Foo.scala
+++ Foo.scala
@@ -1,3 +1,3 @@
-class Foo(s: String) {
+class Foo(private val s: String) {
   def getString = s
 }
```

Reply:
ok, thanks, I'll revert that